<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>343</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEditDisplay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxDigit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButtonDigit7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigit8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigit2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigit4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigit1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigit6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigit5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigit9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigit3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigitStar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigit0">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDigitGrid">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSystem">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton_CmdClear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CmdRedial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>redial</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CmdConf">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>conf</string>
     </property>
    </widget>
    <widget class="QStatusCmdButton" name="pushButton_CmdRecord">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>record</string>
     </property>
    </widget>
    <widget class="QStatusCmdButton" name="pushButton_CmdDND">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>dnd</string>
     </property>
    </widget>
    <widget class="QStatusCmdButton" name="pushButton_CmdMute">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>mute</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CmdHangUp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>hangup</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CmdDial">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>dial</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CmdBackspace">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>&lt;---</string>
     </property>
    </widget>
    <widget class="QStatusCmdButton" name="pushButton_CmdRegister">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>register</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxProgrammButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QProgrammButton" name="programmButton0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>1</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>2</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>5</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>3</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>4</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>9</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton12">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>11</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>6</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>7</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>8</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton11">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>10</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton122">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>21</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton18">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>17</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton20">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>19</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>12</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton23">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>22</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton15">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>14</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton14">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>13</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton21">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>20</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton17">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>16</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton124">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>23</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>18</number>
     </property>
    </widget>
    <widget class="QProgrammButton" name="programmButton16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="numberButton" stdset="0">
      <number>15</number>
     </property>
    </widget>
   </widget>
   <widget class="QProgrammButton" name="programmButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="green" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAaa">
    <property name="title">
     <string>aaa</string>
    </property>
   </widget>
   <addaction name="menuAaa"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QProgrammButton</class>
   <extends>QPushButton</extends>
   <header>programmButton</header>
  </customwidget>
  <customwidget>
   <class>QStatusCmdButton</class>
   <extends>QPushButton</extends>
   <header>statusCmdButton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
